wordpress-mobile/WordPress-Android
# Show privacy settings banner for users in GDPR jurisdictions

Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/18567

To test:
1. Launch (either) app
2. Login with a user from an EU country subject to GDPR (e.g. France)
3. **Expect to see the popup asking your consent**
   | <img width="314" src="https://github.com/wordpress-mobile/WordPress-Android/assets/4588074/29ceb204-e4f1-4dfb-bcc6-1ae17cba27f1" /> |
   | - |
4. **Verify the events** mentioned in [this task](https://github.com/wordpress-mobile/WordPress-Android/issues/18569) are tracked
5. Retry with an user from a non-GDPR country (e.g. US)
6. Expect to **not** see the popup
7. Restart the app after approving or denying consent
8. **Verify the popup is no longer shown**

## Regression Notes
1. Potential unintended areas of impact
   First load of My Site dashboard.

6. What I did to test those areas of impact (or what existing automated tests I relied on)
   n/a

9. What automated tests I added (or what prevented me from doing so)
   The new logic is covered by unit tests.

PR submission checklist:


UI Changes testing checklist:
